UnityProject/Assets/Scripts/Lifecycle/Spawn.cs

```diff
@@ -253,7 +272,8 @@ public static SpawnResult ClientPrefab(string prefabName, Vector3? worldPosition
 	public static SpawnResult ServerClone(GameObject toClone, Vector3? worldPosition = null, Transform parent = null,
 		Quaternion? rotation = null)
 	{
-		return Server(SpawnInfo.Clone(toClone, worldPosition, parent, rotation));
+		return Server(
+			SpawnInfo.Clone(toClone, SpawnDestination.At(toClone)));
 	}
```

Comment:
hm, so this method now ignores `worldPosition`, `parent` and `rotation` parameters and instead assumedly always takes that info from `toClone` gameObject. are you sure this is desired? what if someone wants to clone this object at a different position?